src/test/java/rx/internal/operators/OperatorOnErrorReturnTest.java

```diff
@@ -217,6 +219,33 @@ public void run() {
         }
     }
     
-    
-    
+    @Test
+    public void normalBackpressure() {
+        TestSubscriber<Integer> ts = TestSubscriber.create(0);
+        
+        PublishSubject<Integer> ps = PublishSubject.create();
+        
+        ps.onErrorReturn(new Func1<Throwable, Integer>() {
+            @Override
+            public Integer call(Throwable e) {
+                return 3;
+            }
+        }).subscribe(ts);
+        
+        ts.requestMore(2);
+        
+        ps.onNext(1);
+        ps.onNext(2);
+        ps.onError(new TestException("Forced failure"));
+
+        ts.assertValues(1, 2);
+        ts.assertNoErrors();
+        ts.assertNotCompleted();
+
+        ts.requestMore(2);
```

Comment:
Not a big deal, but it should be `requestMore(1)` if you want to have the same behavior (than the other tests).